docs/en/operations/system-tables/information_schema.md

```diff
@@ -198,6 +199,8 @@ Columns:
     - `FOREIGN TABLE`
     - `LOCAL TEMPORARY`
     - `SYSTEM VIEW`
+- `data_rows` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) — The total
```

Comment:
`table_rows`? See the [docs](https://dev.mysql.com/doc/mysql-infoschema-excerpt/8.0/en/information-schema-tables-table.html)

Reply:
Also fixed, tnx.